xcode/Subconscious/Shared/Components/Common/StoryComboView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  StoryComboView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 24/9/2022.
+//
+
+import SwiftUI
+
+/// A story is a single update within the FeedView
+struct StoryComboView: View {
+    var story: StoryCombo
+    var action: (EntryLink, String) -> Void
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            HStack(spacing: AppTheme.unit) {
+                Text("@bfollington")
+                Text("at")
+                    .foregroundColor(Color.secondary)
+                Text(story.entryA.modified.formatted())
+                    .foregroundColor(Color.secondary)
+                Spacer()
+            }
+            .font(Font(UIFont.appTextSmall))
+            .padding()
+            .frame(height: AppTheme.unit * 11)
+            Divider()
+            VStack(alignment: .leading, spacing: AppTheme.unit4) {
+                Text(story.prompt)
+                
+                Button(
+                    action: {
+                        action(story.entryA.link, story.entryA.linkableTitle)
+                    },
+                    label: {
+                        TranscludeView(entry: story.entryA)
+                    }
+                )
+                .buttonStyle(.plain)
+                
+                Button(
+                    action: {
+                        action(story.entryB.link, story.entryB.linkableTitle)
+                    },
+                    label: {
+                        TranscludeView(entry: story.entryB)
+                    }
+                )
+                .buttonStyle(.plain)
+            }
+            .padding()
+            Divider()
+            HStack {
+                // TODO: make this actually do something
+                // pop open a new note with the other two notes already transcluded in it
+                Button(
+                    action: {
+                        action(story.entryA.link, story.entryA.linkableTitle)
+                    },
+                    label: {
+                        Text("Open")
```

Comment:
Presumably, we'll synthesize a unique link for the new content?

If so, my sense is that "Create" would be a good CTA here, even though it does the same thing under the hood.

Reply:
Yep! My intention was to create a new link (directly at the action dispatch site? Might make sense as shared utility function later?). I wasn't sure what label you'd prefer so I reverted to "Open" and I left it in a "draw the rest of the owl" state.

I'll push things along when I find a few spare hours this week, and/or if we line up a pairing session we'll be able to smash it out then 🙂